File: AtlasWeb/Core/wwwroot/app/NebulaClient/StorageManager/DefinitionManagement/TTQueryParameterDef.ts
import { Guid } from "../../Mscorlib/Guid";
import { TTQueryDef } from "./TTQueryDef";
import { TTDataType } from "../../DataDictionary/TTDataType";

export class TTQueryParameterDef {
    public QueryDef: TTQueryDef;
    public QueryDefID: Guid;
    public ParameterDefID: Guid;
    public Name: string;
    public Description: string;
    public DataType: TTDataType;
    public DataTypeID: Guid;
    public IsArray: boolean;
    public SqlTestValue: string;
    public ToString(): string {
        return null;
    }
}

File: ATLASDEV/AtlasModel/Enterprise/TTQueryParameterDef.cs
using System;

namespace AtlasModel.Enterprise
{
    public partial class TTQueryParameterDef
    {
        public Guid ParameterDefId { get; set; }
        public Guid QueryDefId { get; set; }
        public short OrderNo { get; set; }
        public string Name { get; set; }
        public Guid DataTypeId { get; set; }
        public bool IsArray { get; set; }
        public string SqlTestValue { get; set; }
        public short CheckOutStatus { get; set; }
        public string Description { get; set; }
    }
}